workflow-templates/im-test-selenium-schedule.yml

```
@@ -0,0 +1,177 @@
+# Workflow Code: CosmicFalcon_v1   DO NOT REMOVE
+# Purpose:
+#    Runs Selenium C# tests in GitHub Actions on a schedule.
+#
+# Frequency:
+#    - Workflow template for running Selenium tests on a schedule. This workflow supports the UI Health Check feature
+#
+# Projects to use this Template with:
+#    - Testing projects that use Selenium C#.
+#
+# TODO Prerequisites:
+#    - Add Application Insights secret (APPLICATIONINSIGHTS_CONNECTION_STRING) to every environment you are testing.
+#    - Update your Selenium solution to export test results to Application Insights using the IM.ExportToAppInsights NuGet package. 
+#      Detailed setup guidance can be found here: https://github.com/im-practices/techhub/blob/main/docs/Azure/content/Health-Checks/Selenium-UI-Health-Check.md
+
+name: 🧪 Scheduled Selenium C# Tests
+run-name: 🧪 Run Scheduled Selenium C# Tests
+
+on:
+  schedule: # TODO: Update the schedule to run at the desired frequency. Do not run more than every 30 minutes as to not produce undue load on the system.  
+    - cron: '59 * * * *' # Run Every hour # TODO: Update the frequency of the schedule.
+    # - cron: '1/30 * * * *' # Run Every 30 minutes # TODO: Uncomment this line if you want to run every 30 minutes.
+    # - cron: '1/40 * * * *' # Run Every 40 minutes # TODO: Uncomment this line if you want to run every 40 minutes.
+        
+permissions:
+  id-token: write
+  contents: read
+  actions: read
+
+env:
+  DISPATCH_REF: master
+  SELENIUM_TEST_DIR: 'Selenium' # TODO: Update this to the root directory of the Selenium tests
+  SELENIUM_RESULTS_DIR: 'Selenium/TestResults' # TODO: Update this to the results output directory
+  DOTNET_VERSION: '8.x' # TODO: Specify the .NET SDK version to use
+  TIMEZONE: America/Denver
+  ENVIRONMENT: 'dev' # TODO: Set the default environment
+  BROWSER: 'Chrome' # TODO: Set the default browser
+  TEST_FILTER: 'TestMethod' # TODO: Define test filter if necessary
+
+jobs:
+  cache:
```

```suggestion
  nuget-cache:
```
Same suggestion on other workflow

---

thanks, the job name has been updated.